_posts/atoms/2018-09-27-typography.html

```diff
@@ -2,41 +2,12 @@
 layout: post
 title: Typography
 category: atoms
-description: We stick to [Bootstrap's global settings for typography](https://getbootstrap.com/docs/4.1/content/typography/#global-settings).
-             We use Bootstrap [typography](https://getbootstrap.com/docs/4.1/content/typography/) and [text](https://getbootstrap.com/docs/4.1/utilities/text/) classes when needed.
+description: After some research we decided to stick with system fonts as well known as [Native font stack](https://getbootstrap.com/docs/4.1/content/reboot/#native-font-stack). 
```

Comment:
Missing comma between `research` and `we`.

The part `with system fonts as well known as Native font stack` is a bit weird. I suggest something like:

After some research, we decided to stick with the [Native font stack](https://getbootstrap.com/docs/4.1/content/reboot/#native-font-stack) as we want to rely on system fonts.

Reply:
updated